play-routes/BUILD

```diff
@@ -1,7 +1,46 @@
+load("@bazel_skylib//:bzl_library.bzl", "bzl_library")
 load("@io_bazel_skydoc//stardoc:stardoc.bzl", "stardoc")
+load("@bazel_tools//tools/build_defs/pkg:pkg.bzl", "pkg_tar")
+load("@bazel_skylib//lib:paths.bzl", "paths")
 
-stardoc(
-    name = "play-routes-docs",
-    input = "play-routes.bzl",
-    out = "index.md",
+[
+    stardoc(
+        name = paths.replace_extension(file, "_doc").replace("/", "_"),
+        out = paths.replace_extension(file, ".md").replace("/", "_"),
+        input = file,
+        deps = [
+            "//play-routes:bzl_internal",
+        ],
+    )
+    for file in glob(["**/*.bzl"])
+]
+
+pkg_tar(
+    name = "docs",
+    srcs = [paths.replace_extension(file, ".md").replace("/", "_") for file in glob(["**/*.bzl"])],
+    mode = "0644",
+)
+
+bzl_library(
+    name = "bzl_internal",
+    srcs = [
+        "//play-routes:bzl",
```

Comment:
Can this have `srcs = glob(["**/*.bzl"])` directly? Why do we need the `bzl` filegroup?

Reply:
Good point, it's because it could be scalable to point to `.bzl` in different directories. We can worry about it later.